### PR TITLE
 check  libsqlite3-dev in build.sh , fix copy stockfish_pi and remove setup lichess in postinst 

### PR DESCRIPTION
**Pull request by marcb1309**
Add missing libs libsqlite3-dev to build.sh so that Stockfish can be built on a new image of Raspbian and correct script to add stockfish_pi from an already existing and previous Stockfish build.

Maybe I should check if sources from STOCKFISH_REPO are still up2date to decide if we still need to rebuild even when user selects not to rebuild.

### CODE REVIEW

File: build/postinst [hunk 1]
```diff
@@ -105,8 +106,8 @@ if [[ -e ${SETUP_DIR}/engines/$STOCKFISH ]]
 then
     if [[ -d $CENTAUR_ENGINE ]]
     then
-        mv ${CENTAUR_ENGINE}/stockfish_pi ${CENTAUR_ENGINE}/stockfish_pi.bak
-        mv ${SETUP_DIR}/engines/$STOCKFISH $CENTAUR_ENGINE
+#        mv ${CENTAUR_ENGINE}/stockfish_pi ${CENTAUR_ENGINE}/stockfish_pi.bak
+        mv ${SETUP_DIR}/engines/$STOCKFISH $CENTAUR_ENGINE/stockfish_$STOCKFISH_VERSION
```
Comment (wormstein) [inline build/postinst]: The commented line will backup original SF that comes with centaur. 
And the second one you need to keep the same name (stockfish_pi) so centaur software will find it. This way all games with centaur software will also end up in the database,

Comment (marcb1309) [reply build/postinst]: I reverted my test changes that I did to do some manual testing with different engines on the centaur/engine folder.

**Comment by marcb1309** [comment]
One important thing to mention - I had removed commented the Lichess key during installation since it would break autoinstall!

**Comment by marcb1309** [comment]
Which reminded me to remove ED private Lichess key.

**Comment by wormstein** [comment]
Removing any prompt during install is a good practice anyway.

**Pull request merged**